charts/cluster-addons/values.yaml

```diff
@@ -266,6 +323,26 @@ mellanoxNetworkOperator:
     namespace: network-operator
     values: {}
 
+csiCephfs:
+  enabled: false
+  chart:
+      repo: https://ceph.github.io/csi-charts
+      name: ceph-csi-cephfs
+      version: 3.9.0
+  release:
+    namespace: csi-ceph
+    values: {}
+
+csiNfs:
+  enabled: false
+  chart:
+      repo: https://raw.githubusercontent.com/kubernetes-csi/csi-driver-nfs/master/charts
+      name: csi-driver-nfs
+      version: v4.4.0
+  release:
+    namespace: csi-nfs
+    values: {}
```

Comment:
Can these be nested under a `csi` key please? E.g. `csi.cephfs.*` and `csi.nfs.*`.